alicloud/table_alicloud_rds_backup.go

```diff
@@ -0,0 +1,275 @@
+			{
+				Name:        "backup_mode",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup mode.",
+			},
+			{
+				Name:        "backup_db_names",
+				Type:        proto.ColumnType_STRING,
+				Description: "The name of the database that has been backed up.",
+			},
+			{
+				Name:        "backup_size",
+				Type:        proto.ColumnType_INT,
+				Description: "The size of the backup set. Unit: bytes.",
+			},
+			{
+				Name:        "backup_start_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The beginning of the backup time range. The time is in the yyyy-MM-ddTHH:mm:ssZ format and displayed in UTC.",
+			},
+			{
+				Name:        "backup_end_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The end of the backup time range. The time is in the yyyy-MM-ddTHH:mm:ssZ format and displayed in UTC.",
+			},
+			{
+				Name:        "backup_download_url",
+				Type:        proto.ColumnType_STRING,
+				Description: "The Internet download URL of the backup set. If the download URL is unavailable, this parameter is an empty string.",
+				Transform:   transform.FromField("BackupDownloadURL"),
+			},
+			{
+				Name:        "backup_extraction_status",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup extraction status.",
+			},
+			{
+				Name:        "backup_intranet_download_url",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("BackupIntranetDownloadURL"),
+				Description: "The internal download URL of the backup set.",
+			},
+			{
+				Name:        "backup_method",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup method. Valid values: Snapshot, Physical and Logical.",
+			},
+			{
+				Name:        "backup_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup method.",
+			},
+			{
+				Name:        "backup_initiator",
+				Type:        proto.ColumnType_STRING,
+				Description: "The initiator of the backup task.",
+			},
+			{
+				Name:        "backup_location",
+				Type:        proto.ColumnType_STRING,
+				Description: "The location where backup is available.",
+			},
+			{
+				Name:        "consistent_time",
+				Type:        proto.ColumnType_INT,
+				Description: "The point in time at which the data in the data backup is consistent. ",
+			},
+			{
+				Name:        "copy_only_backup",
+				Type:        proto.ColumnType_STRING,
+				Description: "The backup mode of the data backup. Valid values: 0, 1.",
+			},
+			{
+				Name:        "encryption",
+				Type:        proto.ColumnType_STRING,
+				Description: "The encryption information of the data backup.",
+			},
+			{
+				Name:        "host_instance_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The No. of the instance that generates the data backup. This parameter is used to indicate whether the instance that generates the data backup file is a primary instance or a secondary instance.",
```

Comment:
```suggestion
				Description: "The number of the instance that generates the data backup. This parameter is used to indicate whether the instance that generates the data backup file is a primary instance or a secondary instance.",
```
Is the description correct? The column name suggests that it is the ID of the host instance. How does it specify if the instance is primary or secondary?